src/ontology/uberon-edit.obo

```diff
@@ -223940,16 +223919,12 @@ xref: RO:0002285
 id: develops_from
 name: develops_from
 xref: RO:0002202
-is_transitive: true
 is_a: has_developmental_contribution_from ! has developmental contribution from
```

Comment:
so this isa is not in the import?

Reply:
No, there's another one.

Reply:
problematic.. logical axioms not present in the import need to be investigates. What is the parent in the other one?

Reply:
RO_0002258: developmentally preceded by

Reply:
Alright, here is my suggestion: deal with the definitions for now, but leave the logical axioms that are _not base_. Then we deal with the logical axioms in a separate PR, checking diffs etc.